spec/basic/locators_spec.js

```diff
@@ -1,427 +1,398 @@
-describe('locators', function() {
-  beforeEach(function() {
-    browser.get('index.html#/form');
+describe('locators', () => {
+  beforeEach(async() => {
+    await browser.get('index.html#/form');
   });
 
-  describe('by binding', function() {
-    it('should find an element by binding', function() {
-      var greeting = element(by.binding('greeting'));
+  describe('by binding', () => {
+    it('should find an element by binding', async() => {
+      const greeting = element(by.binding('greeting'));
 
-      expect(greeting.getText()).toEqual('Hiya');
+      expect(await greeting.getText()).toEqual('Hiya');
     });
 
-    it('should allow custom expectations to expect an element', function() {
+    it('should allow custom expectations to expect an element', async() => {
```

Comment:
Not sure what I'm doing... :man_shrugging:

Reply:
This is passing because this is now an xit. This should be turned back on when removing the jasmine-wd node module. I will add this into the main issue to track this.